server/config/development.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Per comment on authController.js, this will eventually be replaced with an `environment/index.js` file.